pkg/util/util.go

```diff
@@ -83,14 +83,8 @@ func IsVFIOVMI(vmi *v1.VirtualMachineInstance) bool {
 	return false
 }
 
-// Check if a VMI spec requests AMD SEV
-func IsSEVVMI(vmi *v1.VirtualMachineInstance) bool {
-	return vmi.Spec.Domain.LaunchSecurity != nil && vmi.Spec.Domain.LaunchSecurity.SEV != nil
-}
-
-// Check if a VMI spec requests SEV with attestation
-func IsSEVAttestationRequested(vmi *v1.VirtualMachineInstance) bool {
-	return IsSEVVMI(vmi) && vmi.Spec.Domain.LaunchSecurity.SEV.Attestation != nil
+func UseLaunchSecurity(vmi *v1.VirtualMachineInstance) bool {
+	return IsSEVVMI(vmi) || IsSecureExecutionVMI(vmi)
```

Comment:
I'm thinking about the idea of using an interface here. However, I'm not entirely sure it's the right fit, since SEV and Secure Execution are conceptually quite different.